File: C/Exercicio35.c
#include <stdio.h>
 
int main(int argc, char *argv[]){
  
  int numeros[10], multi, resul, i;
  
  printf("Digite os 20 numeros: ");
  for(i=1; i<=20; i++){
    scanf("%i", &numeros[i]);
  }
  
  printf("Digite o valor da constante multiplicadora: ");
  scanf("%i", &multi);

  for(i=1; i<=20; ++i){
    printf("\nO valor de %i vezes %i e: ", numeros[i], multi);
    numeros[i] = (numeros[i] * multi);
    printf("%i", numeros[i]);
  }

  return 0;
}

File: Python/Exercicio35.py
numeros = []

for i in range(0, 20, 1):
    x = int(input('Digite um numero: '))
    numeros.append(x)

mult = int(input("Digite a constante multiplicativa: "))

for i in range(0, 20, 1):
    print("O valor de", numeros[i], "vezes", mult, "é:")
    numeros[i] = numeros[i]*mult
    print(numeros[i])